src/tests/components/AuthenticationModal.test.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import AuthenticationModal from '../../components/AuthenticationModal';
+
+describe('AuthenticationModal', () => {
+  it('renders one authentication modal', () => {
+    const component = shallow(<AuthenticationModal debug />);
+      expect(component).toHaveLength(1);
+  
+  });
+  it('div tag displays correctly in "debug" mode', () => {
+    const component = shallow(<AuthenticationModal debug />);
+    expect(component.find('div')).toBeDefined();
+  
+  });
+});
```

Comment:
@MandelaK, also make sure the number of tests you write provides considerable test coverage for the code you have written.